package.json

```diff
@@ -17,7 +17,8 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
-    "prepublish": "yarn run build",
+    "prepublishOnly": "yarn run lint && yarn run test",
```

Comment:
can we remove `test` as it requires extra setup(docker compose)?
I'd like to keep publish as light as possible as it's still a manual operation at the moment(ideally it could be triggered from CI or pipeline),
in most cases, the test should have been executed and verified when PR merged, we don't rely on `publish` to re-check the test.

Reply:
ok